styles/output/statistics-pdf.css

```diff
@@ -1146,6 +1146,13 @@ a[role=doc-noteref] {
   content: target-counter(attr(href, url), page);
 }
 
+.os-formula-review-container {
+  margin-bottom: 1.4rem;
```

Comment:
I have noticed that here and in line 2960 from below this property is repeated.

Reply:
Thank you for noticing that :wink: It'll be cleaned up in the next PR. Shape that adds margin from below will be removed.